src/InputCell.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+export default class InputCell extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: props.row + ' - ' + props.cell,
+      row: parseInt(props.row),
+      cell : parseInt(props.cell),
+      value : 0
+    };
+    this.onInputChangeCallback = props.onChange;
```

Comment:
Why are you doing this?

Reply:
So I can use the callback later on, is there a more appropriate way?